### ISSUE TITLE
No bone-capemgr slots file

### ISSUE THREAD

**Issue by lucalenardi** [brnluiz/bbb-preempt-rt-kernel-patch#2]
I'm not able to find the `slots` file inside `/sys/devices/bone_capemgr.*` so I'm not able to load any cape on the board. Does it work for you?

**Comment by brunoluiz** [comment]
Yes, it is a common issue with Linux versions > 3.8. One thing that you could check is how to compile integrate the capes on device tree (at building, not in run-time). As our time was limited I didn't check it, but one possible solution is showd here: http://hipstercircuits.com/adding-beaglebone-cape-support-to-a-kernel-with-device-tree-in-ubuntu/

If it works for you send some feedback here. Probably, I will study it better for future projects.

In our case we gave up the RT kernels because everytime that was some bug. The 3.8-rt kernels have problems with internet hot-plugging and the 3.8+ have the capemgr issue. We tested PREEMPT kernels and it fulfilled our project needs.

**Comment by lucalenardi** [comment]
Yes It's a mess. I'm trying to find a solution for the Kernel 3.18\* with the latest Debian; to "upgrade" your script it would be sufficient to change the values in `vars.sh`, or there is something else to look at?

**Comment by brnluiz** [comment]
"var.sh" is used to configure the compiler (arm-hf or arm) and which debian image will be used/downloaded. I created this file because, sometimes, I use some of the project scripts in others with different images, kernels and compilers and is easier to change just the vars.sh instead of changing each script.

If you find any fix (as I understood your "upgrade" meaning) to solve this capemgr issue, probably the best thing to do would be create a new script (capemgr-patch.sh or extra-patches.sh) and include it at the "bbb-patch.sh" > start(). Of course, if it work, push the commit to the repository.

**Comment by brunoluiz** [comment]
"var.sh" is used to configure the compiler (arm-hf or arm) and which debian image will be used/downloaded. I created this file because, sometimes, I use some of the project scripts in others with different images, kernels and compilers and is easier to change just the vars.sh instead of changing each script. Maybe you just need to change to "linux_ver" var, but I really don't know how it will work at BBB and with OSADL patches.

If you find any fix to solve this capemgr issue on 3.12 kernel, probably the best thing to do would be create a new script (capemgr-patch.sh or extra-patches.sh) and include it at the "bbb-patch.sh" > start(). Of course, if it work, push the commit to the repository.
